rotkehlchen/chain/ethereum/modules/eth2/utils.py

```diff
@@ -137,6 +139,16 @@ def scrape_validator_daily_stats(
                 column_pos += 1
             elif column_pos == 2:
                 pnl = _parse_fval(column.string, 'income')
+                # if the validator makes profit in the first day beaconchain returns a
+                # profit of deposit + validation reward. We need to subtract the deposit value
+                # to obtain the actual pnl.
+                # Example: https://beaconcha.in/validator/999/stats
+                if pnl > ONE and timestamp == DAY_AFTER_ETH2_GENESIS:
+                    # Since the profit in one day is < 1 ETH get the integer part
+                    # of the profit and subtract it. This covers the case of profit
+                    # and the case of missing attestations on the first day.
+                    pnl -= pnl.to_int(exact=False)
```

Comment:
wait what? Why not just deduct 32?What are you doing here?

Reply:
I had it at -32 ETH and changed it because I short circuited last night with the case of a missing attestation. - 32 is the right thing to do. CHanged it